src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -311,7 +311,7 @@
        * @@returns {String} - A string representing the {@subField}.
        */
       {@context.getApiWrapperName(service)}.prototype.match{@subFieldUpper}From{@fieldPath} =
-          function match{@subFieldUpper}From{@fieldPath}({@fieldParam}) {
+          function({@fieldParam}) {
```

Comment:
nit: you can bring this back to the previous line (`...From{@filedPath} = function({@fieldParam}) {`), since the line would be much shorter.